warplib/ecrecover.cairo

```diff
@@ -26,6 +29,20 @@ func ecrecover_eth{range_check_ptr, bitwise_ptr : BitwiseBuiltin*, keccak_ptr :
 
     let v = v - 27
 
+    let (r_eq_0) = warp_eq256(_r, Uint256(0, 0))
```

Comment:
`_r` is a typo, it should be `r_` same for _s
feel free to change the argument identifiers instead

Reply:
Yeah fixed it already, also added comments.